test/test_torch.py

```diff
@@ -8079,6 +8079,40 @@ def test_complex_flip(self, device, dtype):
                 np_fn = partial(np.flip, axis=flip_dim)
                 self._np_compare_func((torch_fn, np_fn), data, device, dtype)
 
+    def _test_fliplr_flipud(self, funcs, min_dim, max_dim, device, dtype):
+        for dim in range(min_dim, max_dim + 1):
+            shape = self._rand_shape(dim, 5, 10)
+            # Randomly scale the input
+            data = (torch.randn(*shape).to(dtype) * random.randint(50, 100)).tolist()
+            # Use _np_compare_func as it copies the output of np_func,
+            # which takes care of negative strides if present.
+            self._np_compare_func(funcs, data, device, dtype)
```

Comment:
This is going to cause a merge conflict with https://github.com/pytorch/pytorch/pull/39179 as is. That PR is landing now so you should be able to rebase on it shortly.

Reply:
Thanks for the heads up. 

Updated the code.